infra/scripts/docker_build_nncc.sh

```diff
@@ -27,13 +27,13 @@ else
 fi
 
 # prepare tensorflow
-if [ -d $TENSORFLOW_PREFIX ]; then
+if [ -n "$TENSORFLOW_PREFIX" ]; then
```

Comment:
I think the original author wants to check the directory exists also. Maybe we need two condition for empty value and directory check.

Reply:
I think so. How about just add double quotes?
```bash
if [ -d "$TENSORFLOW_PREFIX" ];
```

Reply:
I thought same things, but I removes directory check because we can mount docker volume (not directory).